site/content/docs/api-checks/client-certificates.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Client certificates
+weight: 14
+menu:
+  docs:
+    parent: "API checks"
+---
+
+You can use client certificates to authenticate your API checks to APIs that require mutual TLS (mTLS) authentication, or
+any other authentication scheme where the requester needs to provide a certificate.
+
+Each client certificate is specific to a domain name, e.g. `acme.com` and will be used automatically by any API checks 
+targeting that domain.
+
+> Client certificates are available on the **Enterprise** plan only
+
+## Adding a certificate
+
+Go to the **[Client Certificates](https://app.checklyhq.com/settings/account/client-certificates)** tab on the Account settings screen.
+
+![api checks client certificates](/docs/images/api-checks/client-certificates.png)
+
+1. Click "Add client certificate".
+2. Provide the **host name** for the certificate. You can use wildcard like `*.acme.com`. Under the hood we use [minimatch](https://www.npmjs.com/package/minimatch) 
+3. Select the **certificate file** and **private key** file. Both need to be in `PEM` format.
+4. If your client certificate requires a custom **Certificate Authority**, you can add another `PEM` file that contains one or more concatenated CA certificates.
+5. If you provided a **passphrase** when generating your certificate and private key files, provide it in the passphrase input. You can leave it empty if no passphrase is needed.
+
+> Note that your passphrase will be encrypted at rest in transit and only used at very end when executing your API check. 
+
+## Using a certificate
+
+Client certificates are automatically used by API checks that match the host name of the certificate. There is nothing
+more you need to do. Client certificates are active account wide and cannot be limited to a specific group.
+
+You can add multiple certificate for the same host. In this case we match the certificate by the following rules:
+
+1. The most recently added certificate that matches the host name wins. This goes for both full hostname and ones with wildcards.
+2. A certificate with a full hostname wins over one that matches on a wildcard, e.g. `www.acme.com` wins from `*.acme.com` for the host `www.acme.com`
+
+## Editing a certificate
+
+Client certificates cannot be edited. You can only remove or add client certificates.
+
+## Removing a certificate
+
+You can remove a certificate by going to [Client Certificates](https://app.checklyhq.com/settings/account/client-certificates)
+tab and clicking the **delete** icon. Note that any API checks that require a client certificate will start failing. 
```

Comment:
Lets make this a markdown NOTE tag.

Reply:
what exactly do you mean, the whole paragraph?